src/views/Tasks/CreateTask/index.jsx

```diff
@@ -277,18 +282,22 @@ export default class CreateTask extends Component {
                 value={task || ''}
                 onChange={this.handleTaskChange}
               />
-              <SpeedDial>
-                <SpeedDialAction
-                  requiresAuth
-                  tooltipOpen
-                  icon={<PlusIcon />}
+              <Tooltip
+                className={classes.createIcon}
+                placement="left"
+                title="Create Task"
+              >
+                <Button
+                  variant="fab"
```

Comment:
Could you also add `requiresAuth`, please. We don't user to create tasks when signed out.

```jsx
<Button
  requiresAuth
  variant="fab"
  // ...
```